### PR TITLE
feat: support more zil eco projects

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

File: projects/cctip/project.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: CCTip
+description: A crypto tipping bot currently supporting 5 platforms: Twitter, WeChat, Telegram, Reddit and Discord.
```
Comment (AshIgnYeo) [inline projects/cctip/project.md:3]: wrap the description in quotes since there is the colon in it

File: projects/chainsecurity/project.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: ChainSecurity
+description: A blockchain security and audit expert that helps its clients secure their complex systems.
+categories: false
```
Comment (AshIgnYeo) [inline projects/chainsecurity/project.md:4]: false category

File: projects/coinrule/project.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (AshIgnYeo) [inline projects/coinrule/project.md:1]: check formatting for this file

File: projects/crypto-merch/project.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+name: Crypto Merch
+description: A crypto-focused merchandise e-store that also carries a huge collection of Zilliqa's latest branded merch.
+categories: false
```
Comment (AshIgnYeo) [inline projects/crypto-merch/project.md:4]: false categories

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/cctip/project.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+categories: app
+status: Live
+twitter: @cctip_io
```
Comment (AmritKumar) [inline projects/cctip/project.md:6]:
```suggestion
twitter: https://twitter.com/cctip_io
```

File: projects/chainsecurity/project.md (hunk 2 above)
Comment (AmritKumar) [reply projects/chainsecurity/project.md:4]:
```suggestion
categories: tools
```

File: projects/coinrule/project.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+name: Coinrule
+description: A smart assistant for crypto currency trading, allowing users to take full control of their trading.
+categories: app,tools
+status: Live
+twitter: @CoinRuleHQ
```
Comment (AmritKumar) [inline projects/coinrule/project.md:6]:
```suggestion
twitter: https://twitter.com/CoinRuleHQ
```

File: projects/crypto-merch/project.md (hunk 4 above)
Comment (AmritKumar) [reply projects/crypto-merch/project.md:4]:
```suggestion
categories: app
```

**Review by AshIgnYeo — APPROVED** [review]

**Pull request merged**